TopPlaces/TopPlaces/Controllers/PhotosTableViewController.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+
+#import "PlacesPhotosProvider.h"
+
+NS_ASSUME_NONNULL_BEGIN
+/// Abstract VC class for presenting a list of photos
+@interface PhotosTableViewController : UITableViewController
+
+/// abstract function that populates the photosInfo from some source.
+- (void) fetchPhotos;
+
+/// The list of photo's data in the Flickr format
+/// To be read only once fetchPhotos was invoked.
+@property (strong, nonatomic) NSArray<id<PhotoInfo>> *photosInfo;
```

Comment:
Yes, it seems so:
It's modified in the child classes:
TopPhotosTableViewController::fetchPhotos and 
RecentPhotosTableViewController::fetchPhotos

Can you propose a way to avoid to have it defined as readwrite?